aten/src/ATen/native/native_functions.yaml

```diff
@@ -1672,6 +1672,7 @@
     CPU: resize_
     CUDA: resize_cuda_
     QuantizedCPU: quantized_resize_cpu_
+    Meta: resize_meta_
```

Comment:
This means that we can't make resize_ `structured`, right? You have a check in `gen_structured` to assert that people don't explicitly include their own structured implementations

Reply:
Possible alternative: Just call the meta function (`resize_meta_`) directly rather than through the dispatcher in our codegen. I think that just means that we'll need to call `meta_tensor_from_meta` to convert it to a tensor directly in the codegen as well.

Reply:
aside: I'm not clear on the use case for `out`-variant `meta` functions. If your goal is to find the meta-info of the output tensor, you can just examine `out` argument directly, right?

I could imagine the goal being to take an existing model, stick the `Meta` dispatch key on all the input tensors, and run anything without having to make changes to the model. That's probably it.

Reply:
> This means that we can't make resize_ structured, right? 

This is correct. In fact, we can't make `resize_` structured anyway, because there is no out variant nor purely functional variant (nor would they make sense).

> Possible alternative: Just call the meta function (resize_meta_) directly rather than through the dispatcher in our codegen.

We should do this anyway.

> I could imagine the goal being to take an existing model, stick the Meta dispatch key on all the input tensors, and run everything without having to make changes to the model. That's probably it.

Yep!